DockDoor/Views/Settings/MainSettingsView.swift

```diff
@@ -60,7 +60,7 @@ struct MainSettingsView: View {
             Divider()
 
             HStack {
-                Text("Hover Window Open Delay")
+                Text("Preview Window Open Delay")
```

Comment:
The verbiage "preview" is used for the preview hover actions. Hover window is used throughout the app and codebase to describe what it is, the hover window. Why is this changed?

Reply:
I think for the users "preview" is much clearer than "hover", as it reflects the essence of the window and not its hierarchy, like the window switcher is a "switcher" and not an "overlay"